### PR TITLE
John/testing1

### PR DESCRIPTION
**Pull request by johnwmarrs-coding**
I added many tasts for rspec. I figured I would just merge into develop now to prevent writing tests for features that are changing their styling.

### CODE REVIEW

File: app/controllers/signup_controller.rb [hunk 1]
```diff
@@ -12,6 +12,9 @@ def new
   end
 
   def create
+    puts session[:app_user]
```
Comment (pulchroxloom) [inline app/controllers/signup_controller.rb:15]: Is this code just for testing?

**Review by pulchroxloom — DISMISSED** [review]
Looks more or less good. I flagged some potential unnecessary prints.

File: app/controllers/signup_controller.rb (hunk 1 above)
Comment (johnwmarrs-coding) [reply app/controllers/signup_controller.rb:15]: Yeah that was just me trying to figure out why session[:app_user] was working and session['app_user'] wasn't. I will be sure to remove than line.

**Review by pulchroxloom — APPROVED** [review]
This looks good to me.

**Review by pulchroxloom — APPROVED** [review]
Looks good to me, a lot of tests.

**Pull request merged**